### DISCUSSION TITLE
New Version needed?

### DISCUSSION THREAD

**Discussion by Triskh** [judero01col/GMap.NET#192] [Q&A]
Hello, i'm currently using the latest version in my projekt and until 1 week ago everything works fine. Now my map gets an timeout (OpenstreetMap).

Could it be that OpenstreetMap changed something? An would a new Version of Gmap.net solve the problem?

Pat

**Comment by zgabi** [comment]
Maybe PR #190 solves the issue.

**Reply by Triskh** [reply]
that could be the problem but i where do i add this new url? I use vb.net and in now i only set the mapprovider to openstreetmap.
